src/pages/signUp/signUp.jsx

```diff
@@ -0,0 +1,13 @@
+import { Header } from "../../Components/header/header";
+import { Sidebar } from "../../Components/sidebar/sidebar";
+
+export const SignUpPage = () => {
+  return (
+    <main>
+      <Sidebar />
+      <aside>
+        <Header />
+      </aside>
+    </main>

```

Comment:
Suggestion: Maybe this could be under a single component

Reply:
it is just for intial setup I have to add changes according to the pages